js/main.js

```diff
@@ -0,0 +1,23 @@
+const application = {
+    windowLoad: () => {
+        feather.replace();
+    }
+}
+
+if (
+    document.readyState === "complete" ||
+    (document.readyState !== "loading" && !document.documentElement.doScroll)
+) {
```

Comment:
if (document.readyState === "complete" || document.readyState !== "loading" && !document.documentElement.doScroll) {
Don't think you needed the second set of () but not 100%

Reply:
```js
1 || ( 2 && 3) // if 1 is true OR both 2 &&3 are true
1 || 2 && 3 // if 1 is true and 3 or if 2 is true and three
```
as I understand it at any rate. This bit of code was pulled form [sitepoint](https://www.sitepoint.com/jquery-document-ready-plain-javascript/)